packages/api/src/UserPreferences/index.ts

```diff
@@ -0,0 +1,4 @@
+import { Module } from '@nestjs/common';
+
+@Module({})
+export class UserPreferencesModule {}
```

Comment:
Is this actually used anywhere?

Reply:
Not really, shall I remove it? I still don't completely get how imports/exports work some times... I can get the service from User entity without exporting anything. Maybe the relation between tables has something to do with that?

Reply:
If it's not used it can be deleted. The other user preference stuff is consumed directly by the database module. It appears that modules are just ways of grouping things in nest.